src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
```

Comment:
Love this new hook!

Reply:
right?! I was like... it can't be that easy.. but it is! 😆